docs/docs/dev/policies/mesh-trace.md

```diff
@@ -0,0 +1,340 @@
+# Mesh Trace (beta)
```

Comment:
Would love feedback on how to mark that this policy is not ready to be used & clashes with TrafficTrace

Reply:
I'm fine with this as it matches Kong's policy: https://docs.konghq.com/gateway/latest/stability/#main